consult-dir.el

```diff
@@ -119,10 +132,30 @@ arguments and return a list of directories."
   '(consult-dir--source-bookmark
     consult-dir--source-default
     consult-dir--source-project         ;projectile if available, project.el otherwise
-    consult-dir--source-recentf)
+    consult-dir--source-recentf
+    consult-dir--source-tramp-local)
   "Sources used by `consult-dir--pick'."
   :type '(repeat symbol))
 
+(defun consult-dir--parse-tramp-config (config)
```

Comment:
Rename to `consult-dir--tramp-parse-config`

Reply:
Done :)